manifests/prombench/stateful-set.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: prombench-test-{{ .PR_NUMBER }}
   labels:
     app: prombench-test-{{ .PR_NUMBER }}
+  annotations:
+    prometheus.io/prombench.exit_on_error: 'true'
+    prometheus.io/prombench.wait_on_update: 'true'
```

Comment:
why not have these as cli arg to `resource apply` ? This way the cli help file will show this arg and it will be clear how to use without any need to look inside the code to know that such a functionality exists.

Preferable use it for all apply functions and name `waitOnUpdate` or  otherwise just name it `statefulSet.waitOnUpdate` or something similar to indicate that it is only for statefull set


btw don't we want this behavior in all cases? 
I mean we always want to make sure that the update is completed?

Reply:
waiting is currently only done on `namespace - delete`,  `Deplyment, StatefulSet, DaemonSet - create`, will add waiting on `Deplyment, StatefulSet, DaemonSet - update` too.

This will also get rid of the requirement of `prometheus.io/prombench.wait_on_update: 'true' ` annotation